test/options-manager.js

```diff
@@ -88,6 +88,15 @@ test('buildConfig: settings', t => {
 	t.deepEqual(config.baseConfig.settings, settings);
 });
 
+test('buildConfig: extends', t => {
+	const extendsOption = ['plugin:foo/bar', 'eslint-config-foo-bar', 'foo-bar-two'];
+	const config = manager.buildConfig({extends: extendsOption});
+	const extendsConfig = config.baseConfig.extends;
+	t.is(extendsConfig[extendsConfig.length - 3], 'plugin:foo/bar');
+	t.is(extendsConfig[extendsConfig.length - 2], 'cwd/eslint-config-foo-bar');
+	t.is(extendsConfig[extendsConfig.length - 1], 'cwd/eslint-config-foo-bar-two');
```

Comment:
Actually, would be even nicer to just `Array#slice()` off the items we want an do an `t.deepEqual`.